tcms/telemetry/static/telemetry/js/management/performance.js

```diff
@@ -0,0 +1,142 @@
+// TODO: account for more users
+const colors = [
+  'blue',
+  'red',
+  'gold',
+  'orange',
+  'green',
+  'cyan',
+  'purple',
+  'black',
+  'lightBlue',
+  'lightGreen'
+]
```

Comment:
This should be managed automatically by the charting library (d3 IIRC), no ?

Reply:
yes. removed